docs/specline.rst

```diff
@@ -35,18 +35,21 @@ The base attributes for the SpectralLine class are:
 ================ ================= ========== ========================================
 Property         Variable          Type       Description
 ================ ================= ========== ========================================
+Limits           limits            LineLimits The redshift and limits of the line in redshift, velocity
+                                              (w/r to its redshift) and observed wavelength.
 RA, Dec          attrib['coord']   Coord      astropy.coordinate
-Redshift         attrib['z']       float      Reference redshift
-Redshift sigma   attrib['sig_z']   float      Reference redshift uncertainty
```

Comment:
Would it be worth to have a attrib['z_obs'] and attrib['sig_z_obs'] to allow the user to store a 'measured' redshift within the line? Not particularly strong about this though.